qualtran/bloqs/mod_arithmetic/mod_addition.py

```diff
@@ -339,3 +342,109 @@ def wire_symbol(
         if reg.name == 'y':
             return TextBox(f'y += x*{self.k}')
         raise ValueError(f"Unknown register {reg}")
+
+
+@frozen
+class CModAdd(Bloq):
+    r"""Controlled Modular Addition.
+
+    Implements $\ket{c}\ket{x}\ket{y} \xrightarrow[]{} \ket{c}\ket{x}\ket{(cx+y)\%p}$
```

Comment:
can you just use `\rightarrow`? lots of empty brackets happening in the current arrow

Does `\ket` show up in the notebooks?